.pylintrc

```diff
@@ -1,2 +1,5 @@
 [MESSAGE CONTROL]
-disable=missing-docstring,redefined-outer-name
+disable=invalid-name,missing-docstring,redefined-outer-name
```

Comment:
Added `invalid-name` has it has a 30 character limit, and tests deserve expressive names. (Can it be disabled on `tests` only?)